Dockerfile

```diff
@@ -50,7 +50,7 @@ RUN PYTHONNODEBUGRANGES=1 python -OO -m compileall -b ./python/lib/$runtime/site
 # remove all .py files except ddtrace/contrib/*/__init__.py which are necessary
 # for ddtrace.patch to discover instrumationation packages.
 RUN find ./python/lib/$runtime/site-packages -name \*.py | grep -v ddtrace/contrib | xargs rm -rf
-RUN find ./python/lib/$runtime/site-packages/ddtrace/contrib -name \*.py | grep -v __init__ | xargs rm -rf
+# RUN find ./python/lib/$runtime/site-packages/ddtrace/contrib -name \*.py | grep -v __init__ | xargs rm -rf
```

Comment:
Commenting out this line will have the impact of increasing the layer size significantly. We shouldn't do this.

Reply:
It will also have the impact of increasing cold start time.

Reply:
I'm aware of the related drawbacks. First comment it out to see it's related as the integration tests passed. 
The specific error complained about `logging` module. So maybe we just keep the `logging`'s py files?

Reply:
Updated.